contracts/contracts/vault/OETHVaultCore.sol

```diff
@@ -8,5 +8,19 @@ import { VaultCore } from "./VaultCore.sol";
  * @author Origin Protocol Inc
  */
 contract OETHVaultCore is VaultCore {
+    address public immutable weth;
```

Comment:
nit: a fork test confirming that WETH is set to the correct on-chain address would be nice

Reply:
That would be a good fork test.

Reply:
Agreed. Will add it